### ISSUE TITLE
Font size is too small, page does not respond to zoom well

### ISSUE THREAD

**Issue by karai17** [love2d-community/love-api#103]
Title says it all. On my 27" 1440p monitor at 100% scaling in Windows, the page looks like this:

![image](https://user-images.githubusercontent.com/590332/159726404-341d6727-2110-4ea3-899d-dee288c25f94.png)

Text is hard to read so I try zooming to 200%:

![image](https://user-images.githubusercontent.com/590332/159726970-454a5440-1691-4c5e-8bc0-775797c34780.png)

You can see the sidebar gets cut off on the bottom.

![image](https://user-images.githubusercontent.com/590332/159727180-856246d4-c18d-4590-b771-dcb0bff4547a.png)

Also not sure what's up with the first two items in the list but figured I'd screenshot it too~

If I were to make a suggestion, you could create a couple font size buttons for users to be able to select. Maybe Small: 10pt, Normal: 16pt, and Large: 20, 22, or 24pt?

**Comment by MikuAuahDark** [comment]
I looked at the code to generate the HTML and looks like it misuse the `<details>` and `<summary>` tags. I don't know how to fix it as it breaks other parts of the page when I change it. Furthermore most the position units are specified as pixels, which is the reason why it doesn't scale properly.

A rewritten HTML generator is probably needed but I can't do it without @hahawoo permission as it's their code.

**Comment by hahawoo** [comment]
@MikuAuahDark Thank you for being courteous but my permission is not necessary. :) Anything I have done for this project is public domain, no one needs my permission to change anything, and I also do not claim any ownership or control of any part of this project.

**Comment by MikuAuahDark** [comment]
What about the new page? Does it solve the issue?

**Comment by karai17** [comment]
looks like the sidebar is now fixed when i scale the page up to 200%.

**Comment by MikuAuahDark** [comment]
What about the font size?

**Comment by karai17** [comment]
doesn't really look any different, i still feel the need to scale it up to 200%

**Comment by MikuAuahDark** [comment]
The font size should be slightly increased now. If it's still looks too small to you then I don't think there's much we can do anymore. We have to make compromise between having font size for extraordinary scaling like yours or mobile-friendly.

**Comment by karai17** [comment]
are you using a responsive design? that should be able to adjust the layout
and font sizes based on the screen size and shape. most websites that have
a responsive design allow for mobile all the way to 1080p monitors. i often
have to scale websites to 150% bit this site i need to scale to 200%.

**Comment by MikuAuahDark** [comment]
Yeah. For 1800px, the font size will be 18px. If you have more insights about how to fix this, feel free to check out the `gh-pages` branch and let me know. Thank you.

**Comment by karai17** [comment]
looking through the CSS on the gh-pages branch, I have a few thoughts.

1. there is some inconsistancy between use of absolute and relative units. IMO all font sizes should be based in `pt` rather than `px`, and font scales in `em`. So for example, the body font size could be `12pt` and then the headers could be `1.4em`, or something to that effect.
2. There are some style rules that decrease the font size to `0.9em` and I think all of those should be removed.
3. Font sizes in general are just too small. Using Chrome's viewport developer tool, scaling my viewport down to a mobile device screen to force the media queries to activate, the text feels too small in every size I try. Checking on my own phone as well feels really small.
4. The media query `min-width` cut off points feel rather arbitrary. 1400, 1600, 1800px don't really map cleanly to anything in the real world that I am aware of. IMO a better way to think about this would be to set up queries that target specific form factors. default styles would be for mobile phones, then a query that is designed to trigger on tablets (`min-width: 768px`), then low end laptops (`min-width: 1366px`), then higher end laptops / lower end destops (`min-width: 1920px`), then maybe another rule for higher end desktops (`min-width: 2560px`). Most sites don't bother past 1080p (1920x1080) so i'm used to scaling up pages to 150% to make the page "look" close to 1080p. someone with a 4K monitor or TV would have to scale to 200% to get the same "1080p" look.
